src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/coreperson/CorePersonCprApiMockServer.kt

```diff
@@ -134,7 +133,7 @@ class CorePersonCprApiMockServer : WireMockServer(WIREMOCK_PORT) {
           aResponse()
             .withStatus(status.value())
             .withHeader("Content-Type", "application/json")
-            .withBody(objectMapper.writeValueAsString(if (status == HttpStatus.OK) syncResponse else error)),
+            .withBody(objectMapper.writeValueAsString(if (status == HttpStatus.OK) status else error)),
```

Comment:
looking at the response from this endpoint in person record it is just `OK` so pointless